changelog/10114.improvement.rst

```diff
@@ -0,0 +1 @@
+Replace `atomicwrites` dependency on windows with `os.replace`.
```

Comment:
```suggestion
Replace `atomicwrites <https://github.com/untitaker/python-atomicwrites>`__ dependency on windows with `os.replace`.
```

Reply:
Also perhaps we can consider this `trivial` and backport it to 7.1.x?

Reply:
yep will rename the file and amend when I'm at a computer